fiftyone/core/annotation.py

```diff
@@ -78,3 +82,119 @@ class AnnotationResults(BaseRunResults):
     """Base class for annotation run results."""
 
     pass
+
+
+def compute_annotation_schema(collection, field_name):
```

Comment:
What do you think about calling this "label schema" instead of "annotation schema" for consistency with the existing nomenclature?
https://docs.voxel51.com/user_guide/annotation.html#label-schema

A link to these docs would be nice in this docstring as well 😄

Reply:
How about allowing the user to provide a list of field names as well, in which case the returned schema is a dict in exactly the format described in the link above?